### PR TITLE
REM-596

### PR DESCRIPTION
**Pull request by imtithalhalalo**
page: 'fi-aging'
this is only a table with dynamic columns, the first 3 columns are fixed on the left, total column and row is calculated in the front-end

please check the task image for the added new line

### CODE REVIEW

**Comment by omarZrayka23** [comment]
when i click on  sort icon , i see total row shows as first row

File: src/pages/fi-aging/index.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    return fetchGridData({ _startAt: pagination._startAt || 0, params: filters?.params })
+  }
+
+  const onApply = ({ search, rpbParams }) => {
+    if (!search && rpbParams.length === 0) {
+      clearFilter('params')
+    } else if (!search) {
+      filterBy('params', rpbParams)
+    } else {
+      filterBy('qry', search)
```
Comment (omarZrayka23) [inline src/pages/fi-aging/index.js:151]: u are not need search

File: src/pages/fi-aging/index.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+  const {
+    query: { data },
+    labels: _labels,
+    paginationParameters,
+    filterBy,
+    refetch,
+    access
+  } = useResourceQuery({
+    queryFn: fetchGridData,
```
Comment (omarZrayka23) [inline src/pages/fi-aging/index.js:132]: are u need queryFn?

Comment (imtithalhalalo) [reply src/pages/fi-aging/index.js:132]: yes

File: src/pages/fi-aging/index.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  } = useResourceQuery({
+    endpointId: RGFinancialRepository.FiAging.qry,
+    datasetId: ResourceIds.FIAging,
+    filter: {
+      filterFn: fetchWithFilter
+    }
+  })
+
+  async function fetchWithFilter({ filters, pagination }) {
+    return fetchGridData({ _startAt: pagination._startAt || 0, params: filters?.params })
```
Comment (omarZrayka23) [inline src/pages/fi-aging/index.js:140]: u can put content of  fetchGridData direct here

**Review by omarZrayka23 — APPROVED** [review]

**Pull request merged**